### PR TITLE
Install MLOps Profile with Weave GitOps Blog 

### LINKED ISSUE #263: Create a tutorials section in to documentation
@Chanwit was so nice as to provide us with a lovely tutorial of him setting up the very first user profile.

He documented this here https://docs.google.com/document/d/1fGjVDloLYmJDGbzYAPsAKE62ukKzSvM1fXGIvql4k0g/edit

I think we should add this as a tutorial, or a blog post or something like that on our documentation for https://profiles.dev.

### PR DESCRIPTION
**Pull request by Himangini**
### Description
created blog post based on chanwit's doc for installing MLOps profile
Closes #263 

_NOTE_ I need the screenshot image to add to the blog post

### Checklist
- [ ] Added tests that cover your change
- [ ] Added/modified documentation as required (such as the `README.md` (diagrams, usage, roadmap etc), and anything in `examples/`)
- [x] Made sure the title of the PR is a good description that can go into the release notes
- [x] Added a `kind` label to the PR (e.g. `kind/feature`)

### CODE REVIEW

File: userdocs/profiles.dev/blog/2021-08-04-MLOps-Profile-with-Weave-GitOps.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+3. Check that you already installed both Weave GitOps Core and Profiles CLI, then we install GitOps runtime to our cluster using the following command.
+```
+wego gitops install
+```
+
+5. After that, we install the ProfileManager controller using the pctl prepare command. 
+Please note that we need to tell it that our GitOps runtime is running in the wego-system namespace.
+
+```
+pctl prepare --flux-namespace=wego-system
```
Comment (Skarlso) [inline userdocs/profiles.dev/blog/2021-08-04-MLOps-Profile-with-Weave-GitOps.md:58]: This is now `pctl bootstrap` but I think you're aware of that? :D Just leaving this as a todo :)

Comment (aclevername) [reply userdocs/profiles.dev/blog/2021-08-04-MLOps-Profile-with-Weave-GitOps.md:58]: no! its `pctl install` :smile:

Comment (Skarlso) [reply userdocs/profiles.dev/blog/2021-08-04-MLOps-Profile-with-Weave-GitOps.md:58]: Uh. :D

Comment (Himangini) [reply userdocs/profiles.dev/blog/2021-08-04-MLOps-Profile-with-Weave-GitOps.md:58]: cool i'll add the changes :)

**Comment by chanwit** [comment]
@Himangini  Here's a screenshot of pods listing after the mlops profile got installed.

mlops_profile_pod_listing

File: userdocs/profiles.dev/blog/2021-08-04-MLOps-Profile-with-Weave-GitOps.md (hunk 1 above)
Comment (Himangini) [reply userdocs/profiles.dev/blog/2021-08-04-MLOps-Profile-with-Weave-GitOps.md]: done

File: userdocs/profiles.dev/blog/2021-08-04-MLOps-Profile-with-Weave-GitOps.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
+---
+slug: mlops-profile
+title: Install MLOps Profile with Weave GitOps
+author: Chanwit Kaewkasi
+author_title: DX Engineer, WeaveWorks
+tags: [mlops, profile, gitops]
+---
+
+“_Profiles_” is a GitOps-native package management system, which allows you to apply a Profile to add a set of capabilities to your GitOps-enabled cluster.  Here’s a quick tutorial on how to provision a cluster, install Weave GitOps and apply the MLOps profile to enable Kubeflow Pipeline for your cluster. To try this setup, we strongly recommend that you use KinD as a local cluster on your laptop.
```
Comment (Skarlso) [inline userdocs/profiles.dev/blog/2021-08-04-MLOps-Profile-with-Weave-GitOps.md:9]:
```suggestion
“_Profiles_” is a GitOps-native package management system, which allows you to apply a Profile to add a set of capabilities to your GitOps-enabled cluster.  Here’s a quick tutorial on how to provision a cluster, install Weave GitOps and apply the MLOps profile to enable Kubeflow Pipeline for your cluster. To try this setup, we strongly recommend that you use Kind as a local cluster on your laptop.
```

File: userdocs/profiles.dev/blog/2021-08-04-MLOps-Profile-with-Weave-GitOps.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+First, you need Weave GitOps Core. Please download it from [here](https://docs.gitops.weave.works/docs/installation/)
+
+```
+curl -L "https://github.com/weaveworks/weave-gitops/releases/download/v0.2.2/wego-$(uname)-$(uname -m)" -o wego
+chmod +x wego
+sudo mv ./wego /usr/local/bin/wego
+wego version
+```
+
+Second, you also need Profiles CLI. You can download the CLI from [here](https://github.com/weaveworks/pctl/releases)
```
Comment (Skarlso) [inline userdocs/profiles.dev/blog/2021-08-04-MLOps-Profile-with-Weave-GitOps.md:22]:
```suggestion
Second, you also need Profiles CLI (pctl from here on). You can download the CLI from [here](https://github.com/weaveworks/pctl/releases)
```

**Review by Skarlso — APPROVED** [review]

File: userdocs/profiles.dev/blog/2021-08-04-MLOps-Profile-with-Weave-GitOps.md (hunk 2 above)
Comment (Himangini) [reply userdocs/profiles.dev/blog/2021-08-04-MLOps-Profile-with-Weave-GitOps.md]: fixed

**Pull request merged**